lib/src/workmanager.dart

```diff
@@ -256,8 +261,5 @@ class JsonMapperHelper {
   }
 
   static String _enumToString(final dynamic enumeration) =>
-      enumeration
-          ?.toString()
-          ?.split('.')
-          ?.last;
+      enumeration?.toString()?.split('.')?.last;
```

Comment:
Explanation for this (and other) formatting changes: I usually format a file before saving - internally that is using dartfmt. Let me know if that is unacceptable here.

Reply:
On the contrary, it's a welcome change 🙂